src/test/java/com/wikia/webdriver/testcases/mercurytests/HTMLTitleTests.java

```diff
@@ -122,13 +132,63 @@ private void init() {
   public void mercury_htmlTitleSet() {
     for (String[] testCase : testCases) {
       String testUrl = urlBuilder.appendQueryStringToURL
-              (new Page(testCase[0], testCase[1]).getUrl(),
-                      "cb=" + DateTime.now().getMillis());
+          (new Page(testCase[0], testCase[1]).getUrl(),
+           "cb=" + DateTime.now().getMillis());
 
       navigate.toUrl(testUrl);
       String actualTitle = head.getDocumentTitle();
 
       Assertion.assertEquals(actualTitle, testCase[2]);
     }
   }
+
+  @Test(groups = "seo_guard")
+  public void seo_guard() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)